ci/argocd/templates/octant-application.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     namespace: $DEPLOYMENT_ID
   sources:
     - repoURL: 'https://gitlab.com/api/v4/projects/48137258/packages/helm/devel'
-      targetRevision: 0.2.51
+      targetRevision: 0.2.53
```

Comment:
Should this be updated in this PR?

Reply:
This PR needs new env vars to function. This is why this change was added by Michał.